tekton/images/test-runner/Dockerfile

```diff
@@ -35,6 +35,10 @@ RUN apt update && apt install -y --no-install-recommends \
     openssh-client \
     pkg-config \
     procps \
+    python2 \
```

Comment:
python2 is already installed on the image, but this doesn't harm

Reply:
It is not :stuck_out_tongue: (except if it's already install on the base debian image which I doubt)